README.md

```diff
@@ -60,16 +60,16 @@ Options:
 
 Examples:
   postcss input.css -o output.css                       Basic usage
-  postcss src/**/*.css --base src --dir bild            Glob Pattern & output
+  postcss src/**/*.css --base src --dir build           Glob Pattern & output
   cat input.css | postcss -u autoprefixer > output.css  Piping input & output
 
 If no input files are passed, it reads from stdin. If neither -o, --dir, or
 --replace is passed, it writes to stdout.
 
 If there are multiple input files, the --dir or --replace option must be passed.
 
-Input files may contain globs. If you pass an input directory, it will process
-all files in the directory and any subdirectories.
+Input files may contain globs (ie: src/**/*.css). If you pass an input directory, it will process
+all files in the directory and any subdirectories, respecting the glob pattern.
```

Comment:
Please double-check how the CLI actually outputs this, I'm pretty sure it wraps the lines ~80 chars.

Reply:
ah, yes I was unaware that it was defaulted to 80 chars.  I assumed that was just my screen.

Reply:
![screen shot 2018-04-27 at 9 04 52 am](https://user-images.githubusercontent.com/5241186/39363936-16fc7134-49fa-11e8-8c03-5f635bb552e8.png)

Does this look ok? if so, window is set to 80 columns.

Reply:
@RyanZim  ☝️ that is how it wraps at 80 columns/chars

Reply:
LGTM